pkg/installation/installation.go

```diff
@@ -405,7 +412,7 @@ func (i *Installation) waitForInstaller() error {
 	}
 }
 
-func (i *Installation) buildResult() (*Result, error) {
+func (i *Installation) buildResult(duration time.Duration) (*Result, error) {
```

Comment:
I am wondering if this duration and all the time measuring logic around it should be inside the `Installation`type 🤔 

But it is perfectly fine as is.